src/styles/nav.scss

```diff
@@ -0,0 +1,67 @@
+@import "./variables.scss";
+
+.header {
+  font-size: $font-size-big;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 0 50px;
+  align-items: center;
+  width: 100%;
+}
+
+.logoHolder {
+  position: relative;
+  max-height: 40px;
+  max-width: 40px;
+}
+
+.logo {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  @include delete-margin-padding();
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  color: $text-color-primary;
+  font-weight: $font-weight-mid;
+  text-decoration: none;
+  margin-right: 0;
+  margin-left: 20px;
+  height: 60px;
+  /* Readme said to not use uneven numbers, this is a workaround
+     And despite what the checklist says, the distance between the
+     content on Figma IS 22.5 */
+  padding: calc(45px / 2) 0;
```

Comment:
Try to center it with flexbox

Reply:
The disadvantage of using that is the clickable area becomes only the text itself, making it more difficult to click on the nav

Reply:
It actually stays the same, so you dont have to worry about that
![image](https://user-images.githubusercontent.com/71132144/225567984-fb94a7c5-02d3-48a3-9718-1fafe8aad001.png)

Reply:
Then I think I'm doing it wrong, because when I do it it ends up reducing the clickable space to just the text
Could I get some pointers on how I should go about this?